src/components/dropdown-menu/DropdownMenuItemLink.tsx

```diff
@@ -38,18 +39,27 @@ export function DropdownMenuItemLink({
 
   const isExternal = isExternalProp || toAsString.startsWith('http');
 
+  const externalSuffix = isExternal ? <StyledIconLinkExternal /> : undefined;
+
+  let composedSuffix = suffix ?? externalSuffix;
+
+  if (suffix && externalSuffix) {
+    composedSuffix = (
+      <span className="sw-flex sw-gap-2 sw-items-center">
```

Comment:
We don't have tailwind in echoes

Reply:
hm how come it's working, then? :thinking: Because it's using the consumer's tailwind?

Reply:
yeah, it works because those classes are used in the product so they are included, but it doesn't work in storybook for instance (and it means the consumer must include tailwind with the `sw-` prefix configured)

Reply:
changed to styled component